tests/x11/yast2_lan_restart.pm

```diff
@@ -90,4 +91,8 @@ sub post_fail_hook {
     $self->SUPER::post_fail_hook;
 }
 
+sub test_flags {
+    return {always_rollback => 1};    # Should only affect backends that have snapshot feature
```

Comment:
this comment reminds me times when we had short help with flags description in every `test_flags` call. IMO the fact that flag only affects appropriate backends is part of flag functionality and should be defined in documentation and not in every flag usage

Reply:
It is defined there, as I've introduced it. I will remove the comment =)